homeassistant/components/climate/oem.py

```diff
@@ -0,0 +1,139 @@
+"""
+OpenEnergyMonitor Thermostat Support.
+
+This provides a climate component for the ESP8266 based thermostat sold by
+OpenEnergyMonitor.
```

Comment:
One more thing I just noticed, lets make the documentation file "climate.oem" to match the platform file name.